src/main/java/cinema/exceptions/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package cinema.exceptions;
+
+public class AuthenticationException extends RuntimeException {
```

Comment:
```suggestion
public class AuthenticationException extends Exception {
```
не зміг залогінитися - це не фатально, тому рантайм не потрібно

Reply:
чому не змінила? не прокоментувала?

Reply:
Не помітила :) вже змінила, дякую. Дійсно, помилка не критична, не треба рантайм